the-heist.html

```diff
@@ -1,29 +1,7 @@
-@@include('./partials/header.html')
-
-<div class="section section-with-top">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-
-        <h1>The Heist</h1>
-
-        <!-- Start Gamevy Embed code -->
-        <div class="game-embed"
-             data-gamevy-embed
-             data-operator="gamevy"
-             data-version="1"
-             data-currency="GBP"
-             data-language="en-GB"
-             data-environment="test"
-             data-mode="fun"></div>
-        <script src="https://70234542b69db413cdd4-ec3382d217bd165415e45cc0dc163fa1.ssl.cf3.rackcdn.com/embed.js"></script>
-        <!-- End Gamevy Embed code -->
-
-        <a role="button" class="btn btn-primary pull-right" href="https://70234542b69db413cdd4-ec3382d217bd165415e45cc0dc163fa1.ssl.cf3.rackcdn.com/index.html?env=test&amp;mode=fun&amp;operator=gamevy">Play in full screen</a>
-
-      </div>
-    </div>
-  </div>
-</div>
-
-@@include('./partials/footer.html')
+<!doctype html>
+<html lang="en">
+<head>
+  <meta http-equiv="refresh" content="0; url=/games/the-heist.html">
+</head>
+<body></body>
+</html>
```

Comment:
I've just had a much better idea. We don't need that /games/folder at all, let's get rid of it.